typescript/package.json

```diff
@@ -15,6 +15,6 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "projectmanager-sdk": "^98.0.2363"
```

Comment:
Should it be that version or the latest we just uploaded (`102.0.2949`)?

Reply:
This version number is the latest NPMJS build - https://www.npmjs.com/package/projectmanager-sdk

We only published a new C# because it was affected by the capitalization issue.  We'll redo the other SDKs at the next cycle.